frappe/email/queue.py

```diff
@@ -170,43 +156,20 @@ def get_emails_sent_this_month():
 	return frappe.db.sql("""select count(name) from `tabEmail Queue` where
 		status='Sent' and MONTH(creation)=MONTH(CURDATE())""")[0][0]
 
-def get_unsubscribe_link(reference_doctype, reference_name,
-	email, recipients, expose_recipients, show_as_cc,
-	unsubscribe_method, unsubscribe_params, unsubscribe_message):
-
-	email_sent_to = recipients if expose_recipients else [email]
-	email_sent_cc = ", ".join([e for e in email_sent_to if e in show_as_cc])
-	email_sent_to = ", ".join([e for e in email_sent_to if e not in show_as_cc])
-
-	if email_sent_cc:
-		email_sent_message = _("This email was sent to {0} and copied to {1}").format(email_sent_to, email_sent_cc)
-	else:
```

Comment:
can we also have these messages when the mail gets build again? Should not be too hard (?) Always good to know who else was marked on the mail.